components/UploadedFilePreview.js

```diff
@@ -111,6 +112,15 @@ const UploadedFilePreview = ({ isPrivate, isLoading, isDownloading, url, size, a
     );
   } else if (!url) {
     content = <FileText color="#dcdee0" size="60%" />;
+  } else if (isText) {
+    const icon = <FileTextIcon color="#dcdee0" size="60%" />;
+    content = url ? (
+      <StyledLink href={url} key={url} textAlign="center">
```

Comment:
I would open that in a new tab, since it's an external link

```suggestion
      <StyledLink href={url} key={url} textAlign="center" openInNewTab>
```